tensorboard/webapp/metrics/store/metrics_reducers.ts

```diff
@@ -619,6 +619,19 @@ const reducer = createReducer(
       cardStateMap: nextcardStateMap,
     };
   }),
+  on(actions.metricsCardFullSizeToggled, (state, {cardId}) => {
+    const nextcardStateMap = {...state.cardStateMap};
+    nextcardStateMap[cardId] = {
+      ...nextcardStateMap[cardId],
+      fullWidth: !nextcardStateMap[cardId]?.fullWidth,
+      tableExpanded: !nextcardStateMap[cardId]?.fullWidth,
```

Comment:
I would prefer if setting a card to not full width didn't collapse the table (that is intentionally how it works  today).

Reply:
hmmm my intuition is the opposite. If a user wants to leave the full size mode, more spaces for viewing other cards is better. I would leave it as now. We can always update it when we got any complains.